src/components/related-content/_macro.njk

```diff
@@ -16,7 +16,7 @@
                     <nav class="ons-related-content__navigation" aria-labelledby="{{ row.id }}">
                         {{
                             onsList({
-                                "variants": 'bare',
+                                "variants": row.variants if row.variants else 'bare',
```

Comment:
If variants isn't set and this defaults to bare then the bare class will again be set twice. You can check this by removing the variants param from the related links example

Reply:
I have addressed this